examples/proxy/envelope-encryption/base/proxy/proxy-deployment.yaml

```diff
@@ -0,0 +1,48 @@
+#
+# Copyright Kroxylicious Authors.
+#
+# Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+#
+
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: proxy
+  labels:
+    app: proxy
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: proxy
+  template:
+    metadata:
+      labels:
+        app: proxy
+    spec:
+      containers:
+      - name: proxy
+        image: quay.io/kroxylicious/kroxylicious-developer:0.5.0-SNAPSHOT
```

Comment:
Use downstream

Reply:
@k-wall is the plan to simply keep this manually up-to-date for each downstream build?